app/src/main/java/org/simple/clinic/contactpatient/views/CallPatientView.kt

```diff
@@ -147,7 +147,7 @@ class CallPatientView(
     patientAddressTextView.text = patientDetails.patientAddress
     registeredFacilityTextView.text = patientDetails.registeredFacility
     diagnosisTextView.text = diagnosis
-    lastVisitedTextView.text = dateTimeFormatter.format(patientDetails.lastVisited.toLocalDateAtZone(userClock.zone))
+    lastVisitedTextView.text = dateTimeFormatter.format(patientDetails.lastVisited?.toLocalDateAtZone(userClock.zone))
```

Comment:
When last visited is `null`, we should hide that view. Will that be done in a different PR?

Reply:
Yes, doing that in a different PR.